pkg/kubelet/container/runtime.go

```diff
@@ -97,6 +97,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Cadvisor FS Label
+	ImageLabel() string
```

Comment:
I think it makes sense to be part of the runtime, as its not a simple string, but a cadvisor provided label